Userland/Libraries/LibVideo/PlaybackManager.h

```diff
@@ -25,6 +25,12 @@ namespace Video {
 
 class FrameQueueItem {
 public:
+    FrameQueueItem()
+        : m_data(Empty())
+        , m_timestamp(Time::zero())
+    {
+    }
+
     static constexpr Time no_timestamp = Time::min();
```

Comment:
no_timestamp is "min", should the default init for m_timestamp be min() as well?

Reply:
Good catch on this, I also forgot to use the `no_timestamp` variable in the one instance where it was meant to be used 😅 It's only used for errors that occur when requesting a sample from the demuxer, since there won't be a timestamp available at that point so we want to just present the error to the user as soon as it's dequeued.